src/main/java/com/minecolonies/network/messages/BuildToolPlaceMessage.java

```diff
@@ -30,22 +29,19 @@
  *
  * @author Colton
  */
-public class BuildToolPlaceMessage implements IMessage, IMessageHandler<BuildToolPlaceMessage, IMessage>
+public class BuildToolPlaceMessage extends AbstractMessage<BuildToolPlaceMessage, IMessage>
 {
+    /**
+     * Language key for missing hut message
+     */
+    private static final String NO_HUT_IN_INVENTORY = "com.minecolonies.gui.buildtool.nohutininventory";
     private String hutDec;
     private String style;
     private int    rotation;
-
     private BlockPos pos;
-
     private boolean isHut;
 
     /**
-     * Language key for missing hut message
-     */
-    private static final String NO_HUT_IN_INVENTORY = "com.minecolonies.gui.buildtool.nohutininventory";
-
-    /**
      * Empty constructor used when registering the message.
      */
     public BuildToolPlaceMessage()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) It is a good practice to call super() in a constructor [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ACallSuperInConstructor)